### PR TITLE
Fix and move helm base dir 

### PR DESCRIPTION
**Pull request by zhu733756**
**What type of PR is this?**
Fix and move helm base dir.

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #https://github.com/kubeedge/kubeedge/issues/3660

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
helm base dir changed
```

### CODE REVIEW

File: keadm/cmd/keadm/app/cmd/common/types.go [hunk 1]
```diff
@@ -30,15 +30,16 @@ type InitOptions struct {
...
 	KubeConfig       string
+	KubeEdgeVersion  string
 	AdvertiseAddress string
 	Manifests        string
 	Namespace        string
-	CloudcoreImage   string
+	CloudcoreRepo    string
 	CloudcoreTag     string
-	IptablesMgrImage string
+	IptablesMgrRepo  string
```
Comment (fisherxu) [inline keadm/cmd/keadm/app/cmd/common/types.go:42]: These two options haven't exposed in flag?

Comment (zhu733756) [reply keadm/cmd/keadm/app/cmd/common/types.go:42]: Yes. We can just use --set falg to confirm it.

Comment (fisherxu) [reply keadm/cmd/keadm/app/cmd/common/types.go:42]: Using --set flag to set value in chart directly?

Comment (zhu733756) [reply keadm/cmd/keadm/app/cmd/common/types.go:42]: > Using --set flag to set value in chart directly?

Yes. The user can input their own tag or repo.

Comment (fisherxu) [reply keadm/cmd/keadm/app/cmd/common/types.go:42]: Ok, we need more docs. And the options here can  be removed?

Comment (zhu733756) [reply keadm/cmd/keadm/app/cmd/common/types.go]: Yes, i will fix it

Comment (zhu733756) [reply keadm/cmd/keadm/app/cmd/common/types.go]: Add ref: https://github.com/kubeedge/kubeedge/tree/master/charts/cloudcore/README.md

**Review by fisherxu — COMMENTED** [review]
/approve
/assign @zc2638

**Comment by fisherxu** [comment]
Looks good.

**Comment by zc2638** [comment]
The edgemesh directory under the KubeEdge root directory has been explained, here `manifests/charts/edgemesh` should not need to be added again.

**Comment by fisherxu** [comment]
> The edgemesh directory under the KubeEdge root directory has been explained, here `manifests/charts/edgemesh` should not need to be added again.

We can add edgemesh as addon here?

**Comment by zc2638** [comment]
> We can add edgemesh as addon here?

As a plug-in, edgemesh is a network component supported by the KubeEdge community by default. It is a reasonable way to use it as a default supported option during installation, but we also need to ensure that it does not have to be installed.

**Comment by zhu733756** [comment]
> > We can add edgemesh as addon here?
> 
> As a plug-in, edgemesh is a network component supported by the KubeEdge community by default. It is a reasonable way to use it as a default supported option during installation, but we also need to ensure that it does not have to be installed.

use "--profile version=xxx" will install cloudcore chart, use  "--profile edgemesh" will install edgemesh chart if the chart implements here. Currently, we use  "--external-helm-root" to install edgemesh.

**Comment by zc2638** [comment]
Edgemesh itself should have maintenance charts. Can edgemesh be introduced like middlewares (e.g. mysql, redis)?

**Comment by zc2638** [comment]
/lgtm 
Other work can be completed later.

**Pull request merged**